sacc/tracer.py

```diff
@@ -8,7 +8,7 @@
 
 class Tracer(object):
     def __init__ (self, name, type, z, Nz, exp_sample=None, Nz_sigma_logmean=None,
-                  Nz_sigma_logwidth=None, DNz=None):
+                  Nz_sigma_logwidth=None, DNz=None, Mproxy_name=None, Mproxy_min=None, Mproxy_max=None):
```

Comment:
`Mproxy_min` and `Mproxy_max` define the richness bin here, right?

Reply:
I think the idea is that the mass proxy could be anything. It could be richness, or SZ signal, or something else.

Reply:
Tom is right.